app/helpers/application_helper/toolbar/container_images_center.rb

```diff
@@ -35,6 +35,23 @@ class ApplicationHelper::Toolbar::ContainerImagesCenter < ApplicationHelper::Too
           :url_parms => "main_div",
           :enabled   => "false",
           :onwhen    => "1+"),
+        button(
```

Comment:
This looks very similar to the container_image_center.rb, above... it seems very easy to copy/paste and make a mistake.